CONTRIBUTING.md

```diff
@@ -106,13 +106,25 @@ If you feel that we should try new tools or some aspects of the collaboration co
 
 ## The process of writing chapters
 
-- Create a branch and create a markdown file on it
-- Make a pull request to the turning way master branch. The title of this request should have [WIP] meaning work in progress at the start.
+- Fork the repository form the alan turing version if you have not done so already.
+- On the alan turing version create a branch with the same name as the chapter to be written.
+- On your fork create a branch with the same name and create a markdown file on it.
+- Copy the chapter template in the templates directory into the markdown file, and commit.
+- Make a pull request to the turing way version of the chapter branch. The title of this request should have the form "[WIP] Write Chapter_name chapter". WIP indicates the chapter is a Work In Progress and not yet ready for review.
 - On your branch add material to the chapter and commit. The goal of this project is to collate and build on the many good resources already available about good practise in data science. As such this material should primarily be drawn from outside sources. Note the link and (if available) license of the source.
```

Comment:
While your editing the file, could you quickly go through the whole file and apply the one sentence per line rule please?